EpiCommonUtils/Infrastructure/Validation/XHtmlStringAllowedBlockTypes.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using EPiServer;
+using EPiServer.Core;
+using EPiServer.Core.Html.StringParsing;
+using EPiServer.ServiceLocation;
+
+namespace Forte.EpiCommonUtils.Infrastructure.Validation
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    public class XHtmlStringAllowedBlockTypes : ValidationAttribute
+    {
+        private readonly IContentLoader contentLoader = ServiceLocator.Current.GetInstance<IContentLoader>();
```

Comment:
you use `private readonly IContentLoader contentLoader = ServiceLocator.Current.GetInstance<IContentLoader>();` here and `private Injected<IContentLoader> _contentLoader;` in other attribute. Maybe we can be consistent here?

Reply:
What would be preferred one?

Reply:
I would vote for Injected but it's not a strong prefference. Consistency is primary goal